src/book.js

```diff
@@ -481,6 +490,13 @@ export class Book {
                 currPage.drawPage(page, { y: pos.y, x: pos.x, xScale: pos.sx, yScale: pos.sy, rotate: degrees(rot) });
             }
         });
+        block.forEach((page, i) => {
```

Comment:
Is there a reason this can't be combined with the `forEach` above it?

Reply:
yes- the pages can overlap and the spine-marks can be covered 
This intentionally draws them after we've finished rendering the pages